src/ProjectTemplates/Web.ProjectTemplates/content/EmptyBlazorServerWeb-CSharp/.template.config/dotnetcli.host.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Have you had a chance to test out these CLI options?

Reply:
Yes, these CLI options are working.